templates/components-custom/ec-component-card/card-thumbnail.html.twig

```diff
@@ -0,0 +1,148 @@
+{% spaceless %}
```

Comment:
Cann you add a comment explaining the need for this template? Also add a @see tag pointing to the original template it was based on.

Reply:
True.

Reply:
Pushed.